msal/tests/Test.MSAL.NET.Unit/CacheTests/TokenCacheTests.cs

```diff
@@ -57,7 +58,9 @@ public class TokenCacheTests
         [TestInitialize]
         public void TestInitialize()
         {
+            new TestPlatformInformation();
```

Comment:
>  new TestPlatformInformation(); [](start = 10, length = 32)

are these two (platforminformation and testlogger) really needed for every test? would it be enough just to initialize for the entire test suite?